src/orders/service/model/GetOrdersByRequestFunctionDefinition.ts

```diff
@@ -0,0 +1,7 @@
+import { Order, OrdersQueryRequestByBusinessId, OrdersQueryRequestByChannel, OrdersQueryRequestByChannelOrderId }
+  from '../../..';
+
+type GetOrdersByRequestFunctionDefinition =
+  (ordersRequest: OrdersQueryRequestByBusinessId | OrdersQueryRequestByChannel |
+  OrdersQueryRequestByChannelOrderId, orders: Order[], deferred: Q.Deferred<any>, getSinglePage: boolean) => void;
+export default GetOrdersByRequestFunctionDefinition;
```

Comment:
I'm not sure what this model is for but it does not appear to be used anywhere. Please remove it.